health-on-time/src/Pages/AddMedication.js

```diff
@@ -1,9 +1,62 @@
-import React from "react";
+import React, { useState } from "react";
+import SearchMedication from '../Components/SearchMedication';
+import { Container } from 'react-bootstrap';
+import MedSchedulerMain from '../Components/MedScheduler/MedSchedulerMain.js';
+
+
+
+const AddMedication = ({ handleNewPrescription, prescriptions, token }) => {
+  
+  const [chosenMed, setchosenmed] = useState(); // set method passed as prop down to Medication card
+  
+  const [existingPrescription, setExistingPrescription] = useState(null);
+  
+  function cancelOut() {
+    setchosenmed(null);
+    setExistingPrescription(null);
+  }
+
+  // this passed as prop all the way down to MedicationCard. When its onclick is selected, check for existing 
+  // prescription and set state here
+  function setprescriptionexistsflag(medication){
+    if (prescriptions.find(pres => pres.medication.fda_number === medication.appNumber))
```

Comment:
Shouldn't  this  be
```js
if(prescriptions){
setExistingPrescription(true)} 
```
?

Reply:
No because prescriptions is ALL of user.prescriptions. When user chooses a search result -we want to know if any prescription in user.prescriptions has a matching fda number

Reply:
I think check for existing works for now, but I'll be reviewing what happens when a user chooses a search result that they already scheduled.

Reply:
Damn I made fixes to the wrong branch. I'll be checking to make sure changes mentioned above are complete